plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -337,8 +346,9 @@ public function verify_cot_data( $args = array(), $assoc_args = array() ) {
 
 			$order_ids        = $wpdb->get_col(
 				$wpdb->prepare(
-					"SELECT ID FROM $wpdb->posts WHERE post_type = 'shop_order' AND ID > %d ORDER BY ID ASC LIMIT %d",
+					"SELECT ID FROM $wpdb->posts WHERE post_type = 'shop_order' AND ID > %d AND ID < %d ORDER BY ID ASC LIMIT %d",
```

Comment:
Maybe not in the scope of this PR, but it seems like the start and end should be _inclusive_ of the IDs specified, rather than _exclusive_.

Reply:
Makes sense, addressed here https://github.com/woocommerce/woocommerce/pull/37446/commits/99e4ed4963ab0ca86a85e491a97be1bc3bf61880